modules/ROOT/pages/lifecycle.adoc

```diff
@@ -0,0 +1,94 @@
+= Manage Asset Lifecycle States
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+In the software development lifecycle, assets are developed, released, and eventually deprecated, and each Exchange asset version moves through the lifecycle states of `development`, `stable`, and `deprecated`.
+
+== Development State
+
+An asset version in the `development` state is in an iterative process of design and development.
+
+If you republish an asset version in the `development` state with the same groupId, assetId, and version (GAV) and new metadata, the asset version is overwritten, including the following version metadata:
+
+* files
+* dependencies
+* keywords
+
+Changing this information for an asset version in any other state requires creating a new version, which prevents republishing or overwriting the asset version.
+
+Overwriting an asset version also updates the search index.
+
+The Exchange user interface and APIs change some asset version information for asset versions in any state. This information is not changed by overwriting an asset version in the `development` state. This information includes:
```

Comment:
This paragraph doesn't make sense to me, I'd just add that there is information that is not overwritten by republishing an asset in development state, and list it